packages/common/src/core/types/Metrics.ts

```diff
@@ -221,9 +221,20 @@ export enum ExpressionRelationships {
 export interface IMetricDisplayConfig {
   metricId: string;
   displayType: string;
+  visibility?: MetricVisibility;
```

Comment:
Maybe you already have plans to do this, but I'm wondering if we want to expose other explicit (optional) display config properties on this interface.
Actually, I'm taking a closer look at this file and wondering if `IMetricDisplayConfig` should `extend` `IMetricEditorValues` rather than the other way around (which is what is currently happening)? `IMetricEditorValues` should be properties that come directly out of the editor, whereas `IMetricDisplayConfig` could include other additional properties. Am I confused?

Reply:
Yeah, I think you've got it right -- I'll refactor it to the other way around.

I think we could refactor it to have the optional display config properties -- I'll start looking into that in the create/edit PRs